rosidl_generator_tests/CMakeLists.txt

```diff
@@ -0,0 +1,113 @@
+cmake_minimum_required(VERSION 3.8)
+project(rosidl_generator_tests)
```

Comment:
why is this new package needed?

Reply:
This removes a dependency loop between rosidl_cmake and the generators 

Reference: https://github.com/ros2/rosidl/pull/701